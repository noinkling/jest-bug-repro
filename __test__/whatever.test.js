describe(`outer`, () => {

  describe(`inner one`, () => {
    it(`test one`, async () => {
      expect('foo').toMatchSnapshot();
    });
  });

  describe(`inner two`, () => {
    it(`test two`, async () => {
      expect('bar').toMatchSnapshot();
    });
  });

  describe(`inner three`, () => {
    it(`test three`, async () => {
      expect('baz').toMatchSnapshot();
    });
  });

});
