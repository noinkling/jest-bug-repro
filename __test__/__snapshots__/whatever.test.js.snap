// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outer inner one test one 1`] = `1`;

exports[`outer inner three test three 1`] = `3`;

exports[`outer inner two test two 1`] = `2`;
